src/background/vpncontroller/states.js

```diff
@@ -212,7 +212,7 @@ export class vpnStatusResponse {
       available: false,
       url: "https://localhost:8080",
     },
-    version: "2.25.0",
+    version: "2.99.0",
```

Comment:
what is this now used for?

Reply:
Typinfo - I added this extra to 2.99 so people don't confuse it with something they would ever need to update :)